meta-lmp-base/conf/layer.conf

```diff
@@ -20,6 +20,7 @@ LAYERDEPENDS_meta-lmp-base = " \
     security \
 "
 LAYERSERIES_COMPAT_meta-lmp-base = "kirkstone"
+LAYERSERIES_COMPAT_stm-st-stm32mp:append = " kirkstone"
```

Comment:
meta-lmp-bsp/conf/layer.conf is more apropriate for this imo

Reply:
fixed